dags/utils/gke.py

```diff
@@ -17,13 +20,10 @@ def create_gke_config(
     underscores, and dashes. E.g. owner_label='hwoo', team_label='dataops'
 
     """
-    
+
     cluster_def_dict = {
         "name": name,
-        "masterAuthorizedNetworksConfig":
-            {
-            "enabled": True,
-            },
+        "masterAuthorizedNetworksConfig": {"enabled": not is_dev},
```

Comment:
thanks for this. Should we add a comment as to why we want to toggle this?

Reply:
The container won't run locally without this, since my development machine is not in the authorized network.

Reply:
I meant, adding a comment like "Setting this to false opens the GKE cluster to the world"